README.md

```diff
@@ -11,10 +11,9 @@ The 2 sections I added to build for this project are:
 
 ## Live Demo (if available)
 github repo link 
-[https://github.com/ManarOmari/Project-1-Setup-and-mobile-version-skeleton/pull/1]
+[https://github.com/ManarOmari/Project-1-Setup-and-mobile-version-skeleton/]
```

Comment:
[Optional] I suggest you remove the GitHub repo link because it seems redundant (the visitor would already be in the repo).